components/ctas/Cta.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <a :class="{ button: true, 'button--secondary': secondary }" :href="to">
+  <a :class="{ button: true, 'button--secondary': secondary, 'button--disabled': disabled }" :href="disabled ? null : to">
```

Comment:
Why is this a `a` tag instead of a `button` tag? 🤔 I think using HTML5's `button` tag would make much sense for semantic and a11y reasons. Also, this disabled state would be easier to implement.

Reply:
We are using the `Cta` to redirect people to the proper destinations. In this case, to the application form so it makes sense for it to be a link that sends you to the application form. However, I see your point here and I think it is correct. We can add a region saying the application is closed now instead of adding a `disabled` state for the link. Does it make sense?

Reply:
Do you mean disabling the `button` and showing an explanation of why it is disabled on the bottom? For me, it makes sense if the narrative of the copy is needed. When is this going to happen? Which are the user cases?

Reply:
@lerongil told me he wants to keep the instructions at the same time we make clear the process is closed somehow. I've opened a new issue qiskit/qiskit.org#267 for tracking this. Closing this one, though.